python/setup.py

```diff
@@ -274,6 +276,77 @@ def download_pickle5(pickle5_dir):
                 wzf.close()
 
 
+def patch_isdir():
+    """
+    Python on Windows is having hard times at telling if a symlink is
+    a directory - it can "guess" wrong at times, which bites when
+    finding packages. Replace with a fixed version which unwraps links first.
+    """
+    orig_isdir = os.path.isdir
+
+    def fixed_isdir(path, debug=False):
```

Comment:
```suggestion
    def fixed_isdir(path):
```
This parameter isn't needed, it's a leftover from me fighting with `git` on CI workers on Windows on Azure in conda-forge...